packages/wonder-blocks-button/components/button.js

```diff
@@ -142,6 +142,11 @@ export type SharedProps = {|
     }>>,
     */
 
+    /**
+     * Adds CSS classes to the Button.
+     */
+    className?: string,
```

Comment:
I'd love for us to be able to build-in `Array<string>` support at some point!

Reply:
Yeah, have a `classes` prop would be pretty sweet.